<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 56, 70);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="code_editor">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>591</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 40px;
font: 75 14pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="file_list">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 31, 49);
border-radius: 40px;
color: white;
font: 75 15pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(36, 31, 49);
font: 75 20pt &quot;Dyuthi&quot;;
border-radius: 20px;
border: 5px solid blue;
</string>
    </property>
    <property name="text">
     <string>push</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>410</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(36, 31, 49);
font: 75 20pt &quot;Dyuthi&quot;;
border: 5px solid green;
border-radius: 20px;
</string>
    </property>
    <property name="text">
     <string>run</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="inputs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>141</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 40px;
font: 75 13pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="create_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(36, 31, 49);
font: 75 20pt &quot;Dyuthi&quot;;
border-radius: 20px;
border: 5px solid yellow;
</string>
    </property>
    <property name="text">
     <string>create</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="file_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(36, 31, 49);
border-radius: 25px;
color: white;
font: 75 15pt &quot;Dyuthi&quot;;
border: 7px solid black;
</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(36, 31, 49);
font: 75 20pt &quot;Dyuthi&quot;;
border-radius: 20px;
border: 5px solid red;
</string>
    </property>
    <property name="text">
     <string>remove</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="result">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>421</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(36, 31, 49);
border-radius: 40px;
font: 75 14pt &quot;Dyuthi&quot;;
border: 7px solid black;</string>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
